### PR TITLE
init: add optional pj-rehearse presubmit

### PR DESCRIPTION
**Pull request by droslean**

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 1]
```diff
@@ -137,6 +137,40 @@ presubmits:
...
+          name: job-config
+          readOnly: true
+      volumes:
+      - configMap:
+          name: config
+        name: config
+      - configMap:
+          name: job-config
+        name: job-config
+    trigger: ""
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: Hm how will we trigger the job without the trigger?

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 2]
```diff
@@ -137,6 +137,40 @@ presubmits:
...
+        - /bin/pj-rehearse
+        image: pj-rehearse:latest
+        imagePullPolicy: Always
+        name: ""
+        resources: {}
+        volumeMounts:
+        - mountPath: /etc/config
+          name: config
+          readOnly: true
+        - mountPath: /etc/job-config
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: This is neat

**Pull request merged**